packages/plugin-session-replay-browser/src/typings/session-replay.ts

```diff
@@ -5,4 +5,5 @@ export interface SessionReplayOptions {
   sampleRate?: number;
   privacyConfig?: SessionReplayPrivacyConfig;
   debugMode?: boolean;
+  disableSessionTracking?: boolean;
```

Comment:
`disableSessionTracking` is somewhat confusing imo. It makes me think it would actively disable default tracking sessions.

What about `forceSessionTracking` with a default value of true instead?

Reply:
ooo good points. will make the update in a bit.

Reply:
hmm took a look at this.. it's actually not as easy to have a default value of true. because the param is part of an dictionary/object that's passed in. Also not a js expert, so maybe there is a way. 

What do you think about `disableForcedSessionTracking`? a bit verbose, but more descriptive.

Reply:
I guess i can override the value in the constructor, but logic feels brittle.

Reply:
ok nvm, figured out a clean to do the override in the constructor (was a bit hung up on how undefines/nulls work). the optional param is now `forceSessionTracking` that defaults to true (unless the user explicitly configures it to be false).